test/.ko.yaml

```diff
@@ -0,0 +1 @@
+../.ko.yaml
```

Comment:
This is a symbolic link to `.ko.yaml` so that it uses the overrides from there instead of nothing.